templates/profiler/seo_collector.html.twig

```diff
@@ -14,16 +14,16 @@
            the toolbar panel #}
         <div class="sf-toolbar-info-piece">
             <b>Title</b>
-            <span>{{ collector.title }}</span>
+            <span>{{ collector.title|u.truncate(20, '...', false) }}</span>
```

Comment:
Perhaps put as `title` html property the full content, so you can see it on hover?

Reply:
Wait, [`u`](https://twig.symfony.com/doc/2.x/filters/u.html) is part of `twig/string-extra`. You cannot use it unless requiring it (and it has to be registered by the user by installing the bundle :/

Can't it be elllipsed using CSS instead? Isn't there something similar in the profiler already?

Reply:
Nothing similar in the profiler. I finally used String component in my Data Collector !